pkgs/applications/science/logic/egglog/default.nix

```diff
@@ -5,21 +5,21 @@
 
 rustPlatform.buildRustPackage {
   pname = "egglog";
-  version = "unstable-2023-09-12";
+  version = "unstable-2024-01-26";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

version must start with a number

Reply:
Good catch! Resolved in latest.